### ISSUE TITLE
No way to see server logging without pino-devtools

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-frontend#6410]
### Describe the problem and steps to reproduce it:

Add some server logging. For example:

```diff
diff --git a/src/core/server/base.js b/src/core/server/base.js
index 141cc6234..53b2059c0 100644
--- a/src/core/server/base.js
+++ b/src/core/server/base.js
@@ -264,6 +264,9 @@ function baseServer(
     // Vary the cache on Do Not Track headers.
     res.vary('DNT');
 
+    console.log('--------- LOUD LOG: from console.log');
+    log.info('--------- LOUD LOG: from log.info');
+
     const history = createHistory({ req });
     const { sagaMiddleware, store } = createStore({ history });
 
```

Start the server with `yarn amo:dev`.

### What happened?

The logs do not appear in the console that you started `yarn amo:dev` from.

### What did you expect to happen?

The logs should appear.

### Anything else we should know?

Even though `pino-devtools` shows logging in the devtools panel we should also show them on the console. I was having a hard time debugging https://github.com/mozilla/addons-frontend/issues/6367 because `pino-devtools` doesn't work in that scenario.

@willdurand do you know what's going on? I tried disabling `pino-devtools` but that didn't work (if I did it correctly).

**Comment by willdurand** [comment]
So, there are two things here (I guess):

1. we should use some sort of `tee` to send the logs on both `pino-devtools` and the console
2. `console.log` are missing in the server logs (is that correct?)

In the first case, it is doable and in order to debug #6367, I queried http://localhost:3010/server-logs.json to show the logs 😄 We can do better, so I am going to add this "some sort of `tee` thing".

For (2) however, I need confirmation first. In any case, the patch for (1) would fix this potential issue (as everything would be sent to `stdout` anyway).

**Comment by kumar303** [comment]
Neither of my calls to `console.log()` or `log.info()` were showing up in stdout.

**Comment by willdurand** [comment]
> Neither of my calls to `console.log()` or `log.info()` were showing up in stdout.

Ok, this is fixed by #6419.
